pkg/auth/webapp/session.go

```diff
@@ -58,6 +59,9 @@ type Session struct {
 	// UILocales are the locale to be used to render UI, passed in from OAuth
 	// flow or query parameter.
 	UILocales string `json:"ui_locales,omitempty"`
+
+	// UpdatedAt indicate the session last updated time
+	UpdatedAt time.Time `json:"updated_at,omitempty"`
```

Comment:
I think we need to initialize it when Session is being created.

Reply:
Updated!